cloudify_types/cloudify_types/component/polling.py

```diff
@@ -38,7 +38,7 @@ def blueprint_id_exists(client, blueprint_id):
     """
     blueprint_id_filter = {'id': blueprint_id}
     blueprint = client.blueprints.list(_include=['id'],
-                                       filters=blueprint_id_filter)
+                                       **blueprint_id_filter)
```

Comment:
Why don't you use the filters?

Reply:
I use them, but via the http client you pass them differently them in the resource manager (pass them like I did )